DOCKER/Dockerfile

```diff
@@ -1,17 +1,17 @@
 FROM alpine:3.9
-LABEL maintainer="hello@tendermint.com"
+LABEL maintainer="tenderdash@dash.org"
```

Comment:
I guess we have already something like "developer@dash.org"

Reply:
Adjusted in 3e7d58d